blog/docs/articles/getting-started-blog-p2.md

```diff
@@ -0,0 +1,266 @@
+You can easily interact and navigate Kubernetes resources such as logs for pods with an intuitive explorer tree of Kubernetes resources.
+
+If you are using VSCode, here is the way to set up your [kubernetes plugin](https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-kubernetes-tools&ssr=false#overview):
+
+![](/blog/images/getting-started-blog-series/post2/vscode001.png)
+
+After you have installed the plugin, you will be able to see it here:
+
+![](/blog/images/getting-started-blog-series/post2/vscode002.png)
+
+If you are using Jetbrains IDEs, here is how to install the [kubernetes plugin](https://plugins.jetbrains.com/plugin/10485-kubernetes):
+
+![](/blog/images/getting-started-blog-series/post2/jb001.png)
+
+After you have installed the plugin, you will be able to see it here:
+
+![](/blog/images/getting-started-blog-series/post2/jb002.png)
+
+##### Language Support
+
+VSCode is a versatile text editor, designed to support many programming languages though the Language Server Protocol. By contrast, JetBrains IDEs are language-specific -
+GoLand for Go, Pycharm for Python, and so on. In order for VSCode to provide IDE-style support for a language, you will have to install the required plugin(s) for the language.
+Similarly in JetBrains, you can often install a plugin for a particular language even if the IDE doesn't suppor that language, although sometimes you will have to pay for the
+appropriate license. Since VSCode needs plugins to have IDE features for a language, we recommend that you install the following plugins:
+
+![](/blog/images/getting-started-blog-series/post2/vscode003.png)
+
+![](/blog/images/getting-started-blog-series/post2/vscode004.png)
+
+##### Code Formatting
+
+Go code has a built in style guide and formatter, so naturally you can configure this with both VSCode and JetBrainds IDEs. For VSCode, you can follow the following steps:
+
+![](/blog/images/getting-started-blog-series/post2/vscode005.png)
+
+For JetBrains, you can find more details on how to set it up in [this article](https://www.jetbrains.com/help/idea/integration-with-go-tools.html#gofmt).
+
+## External Setup
+
+Now that you have a working local setup, there are some external things which you will have to set up as well. These can be grouped by how to join the community, how to claim
+issues, and how to make a PR.
+
+### Community
+
+As you are working on your code, you will probably want to discuss your changes with others and ask questions. Knative uses __Slack__ as the primary means of communication,
+along with __a mailing list__. There are also weekly and bi-weekly working group meetings which you can join on zoom where we have discussions. To find the community calendar
+(which includes the working group meeting times and links), as well as the link to the slack and the mailing list, you can [read this article](https://knative.dev/docs/community/#communication-channels).
```

Comment:
Maybe add the link to the communication channels a few sentences before (where you talk about the mailing lists & Slack channels)